web-back/lore/[name].ts

```diff
@@ -29,17 +29,23 @@ app.put(
         const name = req.parameters.name;
         const description = req.body.description as string;
         const title = req.body.title as string;
+        const color = req.body.color as string;
+        const image = req.body.image as string;
+        const thumb = req.body.thumb as string;
```

Comment:
Even if the front-end handles this case, set default values using the `??` operator, as req.body.color & co may be undefined.
It's not a problem for description and title since they are handled by the `if` statement below